<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>scheduler</class>
 <widget class="QWidget" name="scheduler">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1095</width>
    <height>476</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Scan-Jobs</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="1" column="0">
       <widget class="QTabWidget" name="tabWidget">
        <property name="tabPosition">
         <enum>QTabWidget::North</enum>
        </property>
        <property name="tabShape">
         <enum>QTabWidget::Triangular</enum>
        </property>
        <property name="currentIndex">
         <number>0</number>
        </property>
        <property name="iconSize">
         <size>
          <width>22</width>
          <height>22</height>
         </size>
        </property>
        <property name="elideMode">
         <enum>Qt::ElideRight</enum>
        </property>
        <widget class="QWidget" name="dailyTab">
         <attribute name="icon">
          <iconset resource="../resources.qrc">
           <normaloff>:/icons/icons/daily.png</normaloff>:/icons/icons/daily.png</iconset>
         </attribute>
         <attribute name="title">
          <string>Daily Scan-Job</string>
         </attribute>
         <layout class="QGridLayout" name="gridLayout_3">
          <item row="1" column="0">
           <widget class="QLabel" name="dailyScanWithLabel">
            <property name="text">
             <string>Scan with Profile</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QLabel" name="dailyProfileLabel">
            <property name="frameShape">
             <enum>QFrame::Panel</enum>
            </property>
            <property name="frameShadow">
             <enum>QFrame::Sunken</enum>
            </property>
            <property name="text">
             <string>-------------------</string>
            </property>
           </widget>
          </item>
          <item row="1" column="2">
           <widget class="QLabel" name="dailyAtLabel">
            <property name="text">
             <string>every day at</string>
            </property>
           </widget>
          </item>
          <item row="1" column="3">
           <widget class="QTimeEdit" name="dailyTimeEdit">
            <property name="displayFormat">
             <string>hh:mm</string>
            </property>
           </widget>
          </item>
          <item row="1" column="4">
           <widget class="QLabel" name="dailyAClockLabel">
            <property name="text">
             <string>hours.</string>
            </property>
           </widget>
          </item>
          <item row="1" column="5">
           <widget class="QPushButton" name="dailyAddScanJobButton">
            <property name="text">
             <string>add Scan-Job</string>
            </property>
            <property name="icon">
             <iconset resource="../resources.qrc">
              <normaloff>:/icons/icons/scheduleAddButton.png</normaloff>:/icons/icons/scheduleAddButton.png</iconset>
            </property>
            <property name="iconSize">
             <size>
              <width>22</width>
              <height>22</height>
             </size>
            </property>
           </widget>
          </item>
          <item row="0" column="0" colspan="6">
           <widget class="QLabel" name="dailyInfoLabel">
            <property name="styleSheet">
             <string notr="true">padding:5px</string>
            </property>
            <property name="frameShape">
             <enum>QFrame::Panel</enum>
            </property>
            <property name="frameShadow">
             <enum>QFrame::Raised</enum>
            </property>
            <property name="text">
             <string>Create a profile in the Scan - Section and use it here for a timed scan-run. Keep in mind, that only profiles containing selected folder can be uses as a scanning profile for the scheduler. Profiles active in the scheduler can not be changed or removed.</string>
            </property>
            <property name="wordWrap">
             <bool>true</bool>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
        <widget class="QWidget" name="weeklyTab">
         <attribute name="icon">
          <iconset resource="../resources.qrc">
           <normaloff>:/icons/icons/weekly.png</normaloff>:/icons/icons/weekly.png</iconset>
         </attribute>
         <attribute name="title">
          <string>Weekly Scan-Job</string>
         </attribute>
         <layout class="QGridLayout" name="gridLayout_4">
          <item row="1" column="0">
           <widget class="QLabel" name="weeklyScanWithLabel">
            <property name="text">
             <string>Scan with Profile</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QLabel" name="weeklyProfileLabel">
            <property name="frameShape">
             <enum>QFrame::Panel</enum>
            </property>
            <property name="frameShadow">
             <enum>QFrame::Sunken</enum>
            </property>
            <property name="text">
             <string>-------------------</string>
            </property>
           </widget>
          </item>
          <item row="1" column="2">
           <widget class="QLabel" name="weeklyEveryLabel">
            <property name="text">
             <string>every</string>
            </property>
           </widget>
          </item>
          <item row="1" column="3">
           <widget class="QComboBox" name="weeklyDayOfWeekComboBox">
            <item>
             <property name="text">
              <string>Monday</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Tuesday</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Wednesday</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Thursdas</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Friday</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Saturday</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Sunday</string>
             </property>
            </item>
           </widget>
          </item>
          <item row="1" column="4">
           <widget class="QLabel" name="weeklyAtLabel">
            <property name="text">
             <string>at</string>
            </property>
           </widget>
          </item>
          <item row="1" column="5">
           <widget class="QTimeEdit" name="weeklyTimeEdit">
            <property name="displayFormat">
             <string>hh:mm</string>
            </property>
           </widget>
          </item>
          <item row="1" column="6">
           <widget class="QLabel" name="weeklyAClockLabel">
            <property name="text">
             <string>hours.</string>
            </property>
           </widget>
          </item>
          <item row="1" column="7">
           <widget class="QPushButton" name="weeklyAddScanJobButton">
            <property name="text">
             <string>add Scan-Job</string>
            </property>
            <property name="icon">
             <iconset resource="../resources.qrc">
              <normaloff>:/icons/icons/scheduleAddButton.png</normaloff>:/icons/icons/scheduleAddButton.png</iconset>
            </property>
            <property name="iconSize">
             <size>
              <width>22</width>
              <height>22</height>
             </size>
            </property>
           </widget>
          </item>
          <item row="0" column="0" colspan="8">
           <widget class="QLabel" name="weeklyInfoLabel">
            <property name="styleSheet">
             <string notr="true">padding:5px</string>
            </property>
            <property name="frameShape">
             <enum>QFrame::Panel</enum>
            </property>
            <property name="frameShadow">
             <enum>QFrame::Raised</enum>
            </property>
            <property name="text">
             <string>Create a profile in the Scan - Section and use it here for a timed scan-run. Keep in mind, that only profiles containing selected folder can be uses as a scanning profile for the scheduler. Profiles active in the scheduler can not be changed or removed.</string>
            </property>
            <property name="wordWrap">
             <bool>true</bool>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
        <widget class="QWidget" name="monthlyTab">
         <attribute name="icon">
          <iconset resource="../resources.qrc">
           <normaloff>:/icons/icons/monthly.png</normaloff>:/icons/icons/monthly.png</iconset>
         </attribute>
         <attribute name="title">
          <string>Monthly Scan-Job</string>
         </attribute>
         <layout class="QGridLayout" name="gridLayout_5">
          <item row="1" column="0">
           <widget class="QLabel" name="monthlyScanWithLabel">
            <property name="text">
             <string>Scan with Profile</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QLabel" name="monthlyProfileLable">
            <property name="frameShape">
             <enum>QFrame::Panel</enum>
            </property>
            <property name="frameShadow">
             <enum>QFrame::Sunken</enum>
            </property>
            <property name="text">
             <string>-------------------</string>
            </property>
           </widget>
          </item>
          <item row="1" column="2">
           <widget class="QLabel" name="monthlyEveryLabel">
            <property name="text">
             <string>at the</string>
            </property>
           </widget>
          </item>
          <item row="1" column="3">
           <widget class="QSpinBox" name="monthlyDaySpinBox">
            <property name="minimum">
             <number>1</number>
            </property>
            <property name="maximum">
             <number>28</number>
            </property>
           </widget>
          </item>
          <item row="1" column="4">
           <widget class="QLabel" name="monthlyAtLabel">
            <property name="text">
             <string>day at</string>
            </property>
           </widget>
          </item>
          <item row="1" column="5">
           <widget class="QTimeEdit" name="monthlyTimeEdit">
            <property name="displayFormat">
             <string>hh:mm</string>
            </property>
           </widget>
          </item>
          <item row="1" column="6">
           <widget class="QLabel" name="monthlyAClockLabel">
            <property name="text">
             <string>hours.</string>
            </property>
           </widget>
          </item>
          <item row="1" column="7">
           <widget class="QPushButton" name="monthlyAddScanJobButton">
            <property name="text">
             <string>add Scan-Job</string>
            </property>
            <property name="icon">
             <iconset resource="../resources.qrc">
              <normaloff>:/icons/icons/scheduleAddButton.png</normaloff>:/icons/icons/scheduleAddButton.png</iconset>
            </property>
            <property name="iconSize">
             <size>
              <width>22</width>
              <height>22</height>
             </size>
            </property>
           </widget>
          </item>
          <item row="0" column="0" colspan="8">
           <widget class="QLabel" name="monthlyInfoLabel">
            <property name="styleSheet">
             <string notr="true">padding:5px</string>
            </property>
            <property name="frameShape">
             <enum>QFrame::Panel</enum>
            </property>
            <property name="frameShadow">
             <enum>QFrame::Raised</enum>
            </property>
            <property name="text">
             <string>Create a profile in the Scan - Section and use it here for a timed scan-run. Keep in mind, that only profiles containing selected folder can be uses as a scanning profile for the scheduler. Profiles active in the scheduler can not be changed or removed.</string>
            </property>
            <property name="wordWrap">
             <bool>true</bool>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </widget>
      </item>
      <item row="0" column="0" colspan="2">
       <widget class="QTableWidget" name="scanJobTableWidget">
        <property name="editTriggers">
         <set>QAbstractItemView::NoEditTriggers</set>
        </property>
        <property name="showDropIndicator" stdset="0">
         <bool>false</bool>
        </property>
        <property name="wordWrap">
         <bool>false</bool>
        </property>
        <property name="columnCount">
         <number>8</number>
        </property>
        <attribute name="horizontalHeaderCascadingSectionResizes">
         <bool>true</bool>
        </attribute>
        <attribute name="horizontalHeaderShowSortIndicator" stdset="0">
         <bool>true</bool>
        </attribute>
        <attribute name="horizontalHeaderStretchLastSection">
         <bool>true</bool>
        </attribute>
        <attribute name="verticalHeaderVisible">
         <bool>false</bool>
        </attribute>
        <attribute name="verticalHeaderStretchLastSection">
         <bool>false</bool>
        </attribute>
        <column/>
        <column/>
        <column/>
        <column/>
        <column/>
        <column/>
        <column/>
        <column/>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QFrame" name="profileInfoFrame">
        <property name="minimumSize">
         <size>
          <width>400</width>
          <height>0</height>
         </size>
        </property>
        <property name="frameShape">
         <enum>QFrame::WinPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <layout class="QGridLayout" name="gridLayout_8">
         <item row="0" column="0">
          <widget class="QLabel" name="profileInfoHeaderLabel">
           <property name="styleSheet">
            <string notr="true">padding:5px;background:#c0c0c0</string>
           </property>
           <property name="frameShape">
            <enum>QFrame::WinPanel</enum>
           </property>
           <property name="frameShadow">
            <enum>QFrame::Sunken</enum>
           </property>
           <property name="text">
            <string>Available Profiles and Profile Information</string>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <layout class="QHBoxLayout" name="horizontalLayout">
           <item>
            <widget class="QLabel" name="profileLabel">
             <property name="text">
              <string>Select Profile</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QComboBox" name="profileComboBox"/>
           </item>
          </layout>
         </item>
         <item row="2" column="0">
          <widget class="QFrame" name="frame">
           <property name="frameShape">
            <enum>QFrame::StyledPanel</enum>
           </property>
           <property name="frameShadow">
            <enum>QFrame::Raised</enum>
           </property>
           <layout class="QHBoxLayout" name="horizontalLayout_4">
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_2">
              <item>
               <widget class="QLabel" name="targetLabel">
                <property name="text">
                 <string>Target(s)</string>
                </property>
                <property name="alignment">
                 <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLabel" name="targetInfoLabel">
                <property name="styleSheet">
                 <string notr="true"/>
                </property>
                <property name="frameShape">
                 <enum>QFrame::NoFrame</enum>
                </property>
                <property name="frameShadow">
                 <enum>QFrame::Plain</enum>
                </property>
                <property name="text">
                 <string/>
                </property>
                <property name="scaledContents">
                 <bool>true</bool>
                </property>
                <property name="alignment">
                 <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
                </property>
                <property name="wordWrap">
                 <bool>true</bool>
                </property>
               </widget>
              </item>
             </layout>
            </item>
           </layout>
          </widget>
         </item>
         <item row="3" column="0">
          <widget class="QFrame" name="frame_2">
           <property name="frameShape">
            <enum>QFrame::StyledPanel</enum>
           </property>
           <property name="frameShadow">
            <enum>QFrame::Raised</enum>
           </property>
           <layout class="QGridLayout" name="gridLayout_6">
            <item row="0" column="0">
             <layout class="QHBoxLayout" name="horizontalLayout_3">
              <item>
               <widget class="QLabel" name="optionsLabel">
                <property name="text">
                 <string>Options</string>
                </property>
                <property name="alignment">
                 <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLabel" name="optionsInfoLabel">
                <property name="frameShape">
                 <enum>QFrame::NoFrame</enum>
                </property>
                <property name="frameShadow">
                 <enum>QFrame::Plain</enum>
                </property>
                <property name="text">
                 <string/>
                </property>
                <property name="scaledContents">
                 <bool>true</bool>
                </property>
                <property name="alignment">
                 <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
                </property>
                <property name="wordWrap">
                 <bool>true</bool>
                </property>
               </widget>
              </item>
             </layout>
            </item>
           </layout>
          </widget>
         </item>
         <item row="4" column="0">
          <widget class="QFrame" name="frame_3">
           <property name="frameShape">
            <enum>QFrame::StyledPanel</enum>
           </property>
           <property name="frameShadow">
            <enum>QFrame::Raised</enum>
           </property>
           <layout class="QGridLayout" name="gridLayout_7">
            <item row="0" column="0">
             <widget class="QLabel" name="logFileLabel">
              <property name="text">
               <string>Log-File : </string>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
         <item row="5" column="0">
          <spacer name="verticalSpacer">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>34</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../resources.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>monthlyAddScanJobButton</sender>
   <signal>clicked()</signal>
   <receiver>scheduler</receiver>
   <slot>slot_addMonthlyScanJobButtonClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>110</x>
     <y>279</y>
    </hint>
    <hint type="destinationlabel">
     <x>3</x>
     <y>352</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>weeklyAddScanJobButton</sender>
   <signal>clicked()</signal>
   <receiver>scheduler</receiver>
   <slot>slot_addWeeklyScanJobButtonClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>110</x>
     <y>279</y>
    </hint>
    <hint type="destinationlabel">
     <x>7</x>
     <y>347</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>dailyAddScanJobButton</sender>
   <signal>clicked()</signal>
   <receiver>scheduler</receiver>
   <slot>slot_addDailyScanJobButtonClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>517</x>
     <y>389</y>
    </hint>
    <hint type="destinationlabel">
     <x>6</x>
     <y>320</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>profileComboBox</sender>
   <signal>activated(QString)</signal>
   <receiver>scheduler</receiver>
   <slot>slot_profileSelectionChanged()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>1037</x>
     <y>289</y>
    </hint>
    <hint type="destinationlabel">
     <x>1089</x>
     <y>248</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>slot_addMonthlyScanJobButtonClicked()</slot>
  <slot>slot_addDailyScanJobButtonClicked()</slot>
  <slot>slot_addWeeklyScanJobButtonClicked()</slot>
  <slot>slot_profileSelectionChanged()</slot>
 </slots>
</ui>
